Gemfile

```diff
@@ -2,14 +2,41 @@ source "https://rubygems.org"
 
 gemspec
 
-gem 'rspec-its'
+# Temporary hack to troubleshoot what elrayle is seeing
+gem 'rails', '~> 5.0.2'
```

Comment:
I tested this branch with rails 5.0.2 and 5.1.0.  Everything works fine at 5.0.2.  There are incompatible dependencies at 5.1.0 when trying to generate engine_cart.